src/pudl/transform/eia923.py

```diff
@@ -613,6 +613,9 @@ def generation_fuel(eia923_dfs, eia923_transformed_dfs):
     """
     # This needs to be a copy of what we're passed in so we can edit it.
     gen_fuel = eia923_dfs["generation_fuel"].copy()
+    gen_fuel.rename(columns={"eia_sector": "sector_id_eia"}).rename(
```

Comment:
You can give `df.rename()` a dictionary of many columns to rename and it'll do them all at once, rather than needing to chain methods.  Also note that in most cases methods return a new object rather than mutating the dataframe, you you actually need to do something like this, or the change won't persist:

```py
gen_fuel = gen_fuel.rename(
    columns={
        "eia_sector": "sector_id_eia",
        "sector_name": "sector_name_eia",
    }
)
```

Reply:
However, I think the right place to change these names is in the column maps under `src/pudl/package_data/eia923/column_maps/*.csv`  Those CSVs control how the spreadsheet columns get renamed after being read, so that all of the years of data can be concatenated together into a single large dataframe, with standardized column names across all years.

The first column is the name that the column will be given, and the other columns are the `snake_case` versions of the column names as they appear in the spreadsheets. So if you change the first column from `eia_sector` to `sector_id_eia` then the concatenated dataframes which are the inputs to the transform functions will already be named appropriately, and you can just remove the code that would have dropped them (under their old names) and avoid the need to rename the columns in the transforms.